pkgs/development/compilers/rust/rust-src.nix

```
@@ -1,14 +1,25 @@
-{ stdenv, rustc, minimalContent ? true }:
+{ lib, stdenv, rustc, minimalContent ? true }:
 
 stdenv.mkDerivation {
   name = "rust-src";
   src = rustc.src;
   phases = [ "unpackPhase" "installPhase" ];
   installPhase = ''
     mv src $out
-    rm -rf $out/{${if minimalContent
-      then "ci,doc,etc,grammar,llvm-project,llvm-emscripten,rtstartup,rustllvm,test,tools,vendor,stdarch"
-      else "ci,doc,etc,grammar,llvm-project,llvm-emscripten,rtstartup,rustllvm,test,vendor"
-    }}
+    rm -rf $out/{${lib.concatStringsSep "," ([
```

```suggestion
    rm -r $out/{${lib.concatStringsSep "," ([
```
Should it fail if any of the directories changes?

---

@SuperSandro2000 I don't think this list is up to date since the src was reorganized, so I rather deal with that later. Indeed for the rest of the PR, I use `rustLibSrc` instead.